storybookjs/storybook
# CLI: Exclude addon-styling from upgrade

Closes #23749 

## What I did

- Exclude `@storybook/addon-styling` and `@storybook/addon-styling-webpack` from the upgrade

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Install `@storybook/addon-styling`
3. Run `npx storybook@latest upgrade`
4. No warning should throw because addon-styling doesn't match versions with core packages



## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`



### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
